javascript/javascript3/week1/my-own-json.js

```diff
@@ -0,0 +1,6 @@
+const jsonFavouriteFood =
+  '{"List of dinner":["shawrma","fruits","salat"], "list of lanch":["dolma", "rice", "salat"]}';
+console.log(jsonFavouriteFood);
```

Comment:
don't console.log inside Json file.

Always use JSON validator/beautifier like : https://jsonformatter.curiousconcept.com/

Reply:
and its like string becuase you are putting ' before , remove ' 

it should be like : 

{
   "List of dinner":[
      "shawrma",
      "fruits",
      "salat"
   ],
   "list of lanch":[
      "dolma",
      "rice",
      "salat"
   ]
}